### PR TITLE
[ARTEMIS-561] JMS Bridge should display statistics about messages tha…

### PR DESCRIPTION
**Pull request by bayern39**
https://issues.apache.org/jira/browse/ARTEMIS-561

i have added two metrics about jms-bridge.
the attribute of  'message_count' record the number of messages successfully committed.
the attribute of 'aborted_message_count' record the number of messages aborted/rollback.

### CODE REVIEW

File: artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/bridge/JMSBridge.java [hunk 1]
```diff
@@ -90,6 +90,10 @@
 
    boolean isFailed();
 
+   int getMessageConut();
```
Comment (clebertsuconic) [inline artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/bridge/JMSBridge.java]: typo... Count

Comment (clebertsuconic) [inline artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/bridge/JMSBridge.java]: @bayern39  can you fix the typo?

**Comment by bayern39** [comment]
@clebertsuconic  ok,i have fixed  the mistake.i am so sorry that yesterday i have no take a attention in this  when i was after work.

**Comment by clebertsuconic** [comment]
I have renamed those variables to follow the conventions used on the project (no _ on variable names)..

I also changed those attributes to long just in case

**Pull request closed without merging**